pkg/kubelet/cm/cpumanager/policy_options_test.go

```diff
@@ -80,6 +82,12 @@ func TestPolicyBetaOptionsAvailable(t *testing.T) {
 			featureGateEnable: false,
 			expectedAvailable: false,
 		},
+		{
+			option:            AlignBySocketOption,
+			featureGate:       pkgfeatures.CPUManagerPolicyAlphaOptions,
+			featureGateEnable: true,
+			expectedAvailable: true,
+		},
```

Comment:
Can you add one more test case that ensures this is not available if CPUManagerPolicyBetaOptions is set.

Reply:
Fixed!